vllm/default_logging_config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "logger": {
+    "structured_log_file_path": "/var/log/vllm/application_structured.log",
```

Comment:
Feel like writing to a file by default needs a lot more work than this. There are some problems here;

1. /var is not accessible in many environments. For example, I tried this in my env and failed immediately. I recommend you to use `/tmp/vllm`. It's been done this way from ray https://github.com/ray-project/ray/, and it worked pretty well.
2. If you start a new vllm instance, what's going to happen to the previous logs? Is it overwritten? Ideally we should create a directory for each new instance

I'd suggest to not write to a file by default (same as master) and add an example to the doc instead.